src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/BannedAssembliesAnalyzer.cs

```diff
@@ -38,7 +38,10 @@ static void CheckType(ISymbolAnalysisContext context, ITypeSymbol type, ISymbol
                 {
                     if (BannedAssemblies.Contains(type.ContainingAssembly.Name))
                     {
-                        context.ReportDiagnostic(Diagnostic.Create(Descriptors.AZC0014, symbol.Locations.First(), BannedAssembliesMessageArgs), symbol);
+                        if (type.Name != "Utf8JsonReader" && type.Name != "Utf8JsonWriter")
```

Comment:
is there a way to compare fully qualified names (at least with the namespace), not just type names?